views/amp-access/article.html

```diff
@@ -79,7 +83,7 @@
     <main role="main">
     <article>
 
-      <section id="meter-notif" amp-access="access" class="meter-section" data-z-depth="2">
+      <section id="meter-notif" amp-access="access OR error" class="meter-section" data-z-depth="2">
```

Comment:
I that case I'd recommend not adding it in to the content section - I might give the wrong ideas. Having one separate section for it should be enough.